docs/command_line.md

```diff
@@ -67,3 +69,28 @@ pyms encrypt 'mysql+mysqlconnector://important_user:****@localhost/my_schema'
 ```
 
 See [Encrypt/Decrypt Configuration](encrypt_decryt_configuration.md) for more information
+
+## Merge swagger into a single file
+
+Command: 
+
+```bash
+pyms merge-swagger [-h] [-f FILE]
+```
+
+```bash
+optional arguments:
+  -h, --help            show this help message and exit
+  -f FILE, --file FILE  Swagger file path
+```
+
+This command use [prance](https://github.com/jfinkhaeuser/prance) to validate the API specification and generate a single YAML file. Has an optional argument to indicate the main file path of the API specification.
+
+!!! warning
+    You must run first `pip install prance==0.18.2`
```

Comment:
In this doc there's no mention of the additional dependencies that the commands might need, I wouldn't put it here for consistency.